Source/ocdm/playLevel.h

```diff
@@ -0,0 +1,71 @@
+ /*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "Module.h"
+
+using namespace WPEFramework::Core ;
+
+/** Playback protection levels struct. */
+struct PlayLevels {
+
+    class PlayLevelsJSON : public JSON::Container {
+    private:
+        PlayLevelsJSON(const PlayLevelsJSON&) = delete;
+        PlayLevelsJSON& operator=(const PlayLevelsJSON&) = delete;
+
+    public:
+        PlayLevelsJSON()
+            : WPEFramework::Core::JSON::Container()
+            , m_CompressedVideo(0)
```

Comment:
Don't know if this was in the original code but you remember we discussed before? Better to use the default constructor unless you really want to indicate you have a meaningful default value (and that is not the case here)

Reply:
indeed original code